### ISSUE TITLE
Message list component: the message type is always error when parameters specified

### ISSUE THREAD

**Issue by dmytro-ch** [magento/magento2#17700]
### Preconditions
Magento 2.2.5

### Description
Component: ``Magento_Ui/js/model/messageList.js`` component.
The message type is always **"error"** when specifying the ``parameters`` property.

### Steps to reproduce

1. Use message list component (``Magento_Ui/js/model/messageList.js``).
2. Add success message with parameters. E.g. 
```
messageList.addSuccessMessage({message: 'Sample %test message.', parameters: {test: 'Success'}});
```

### Expected result

1. The success message should be shown.
![screen shot 2018-08-20 at 3 24 57 pm](https://user-images.githubusercontent.com/11693779/44340864-daebd500-a48e-11e8-8c7b-11ff024117f5.png)

### Actual result

1. The error message is shown.
![screen shot 2018-08-20 at 3 36 30 pm](https://user-images.githubusercontent.com/11693779/44340868-e0e1b600-a48e-11e8-8d35-c2622dfe2254.png)

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [x] Summary of the issue
- [x] Information on your environment
- [x] Steps to reproduce
- [x] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@dmytro-ch do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [x] yes
- [ ] no

**Comment by ghost** [comment]
@dmytro-ch, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your report. 
The issue has been fixed in magento/magento2#17701 by @dmytro-ch in 2.2-develop branch
Related commit(s):
 - [d438e9b68d111715b09c7252786ac2ec102ba19c](https://github.com/magento/magento2/commit/d438e9b68d111715b09c7252786ac2ec102ba19c)

The fix will be available with the upcoming 2.2.7 release.

**Comment by sidolov** [comment]
Hi @dmytro-ch. Thank you for your report. 
The issue has been fixed in magento/magento2#17702 by @dmytro-ch in 2.1-develop branch
Related commit(s):
 - [831fe2f8f616278d0bb81f82d3f07c74619fd073](https://github.com/magento/magento2/commit/831fe2f8f616278d0bb81f82d3f07c74619fd073)

The fix will be available with the upcoming 2.1.16 release.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your report. 
The issue has been fixed in magento/magento2#17703 by @dmytro-ch in 2.3-develop branch
Related commit(s):
 - [88aa0bd9b77856f2f10a507cb120fd4e9309fa96](https://github.com/magento/magento2/commit/88aa0bd9b77856f2f10a507cb120fd4e9309fa96)

The fix will be available with the upcoming 2.3.0 release.
